src/zino/tasks/reachabletask.py

```diff
@@ -28,6 +28,8 @@ async def run(self):
             if event.reachability != ReachabilityState.NORESPONSE:
                 event.reachability = ReachabilityState.NORESPONSE
                 event.add_log(f"{self.device.name} no-response")
+            event.polladdr = self.device.address
+            event.priority = self.device.priority
```

Comment:
Why are polladdr and priority also added to every event?

Reply:
Because Zino 1 does it, and it was neglected from several of the task plugins.

It could be argued that it doesn't belong in this particular PR, but I discovered it while comparing Zino 1 events to Zino 2 events during this PR, so I threw it in there for good measure :-D